src/ophyd_async/epics/pmac/_pmacTrajectory.py

```diff
@@ -0,0 +1,155 @@
+import time
+
+import numpy as np
+import numpy.typing as npt
+from bluesky.protocols import Flyable, Preparable
+from scanspec.specs import Frames, Path
+
+from ophyd_async.core.async_status import AsyncStatus, WatchableAsyncStatus
+from ophyd_async.core.signal import observe_value
+from ophyd_async.core.utils import WatcherUpdate
+from ophyd_async.epics.motion import Motor
+from ophyd_async.epics.pmac import Pmac
+
+TICK_S = 0.000001
+
+
+class PmacTrajectory(Pmac, Flyable, Preparable):
```

Comment:
Please could you implement this as a `TriggerLogic` to be nested within a `StandardFlyer`? That will remove a bit of the boilerplate around `kickoff` and `complete`